FogCreek/Glitch-Community
# Grizzled lamprey: Add MyStuff button to project pages

## Links
* https://grizzled-lamprey.glitch.me
* https://app.zenhub.com/workspaces/community-5d07d1359dee6059a9688176/issues/fogcreek/glitch-community-work/166
* https://www.notion.so/glitch/My-Stuff-Default-Collection-f9eb3c8196c6431596fba526fc5b91c2#13dadf94dc1b45d492f5939a23cb41f3

## GIF/Screenshots:

![](http://g.recordit.co/EWsskPudJI.gif)

## Changes:
* adds a bookmark button to project pages that shows when the user is logged in (only shows when dev toggle is set)
* moves privacy button below name of project
* truncates project text when it's too long
* I removed all the bookmark assets and re-added them because I was dealing with some weird merge conflicts in .glitch-assets and thought that might help? In any case it seems ok now to me. 

## How To Test:
* on a test account go to https://grizzled-lamprey.glitch.me/secret and enable my stuff
* navigate to a project page
* press the bookmark button (if you don't have a My Stuff collection yet you'll see it creates one for you under the hood and then adds the project to that collection
* see the project in your my stuff collection
* ensure you don't see the bookmark button when you are logged out

## Feedback I'm looking for:
- the animation is looking a little choppy for me and I'm not sure why. If anyone can figure out why that would be cool!
- any other bugs or things you see
